qabelbox/src/androidTest/java/de/qabel/qabelbox/ui/views/FileBrowserFragmentTest.kt

```diff
@@ -56,6 +47,7 @@ class FileBrowserFragmentTest: AbstractUITest() {
     }
 
 
+    @Ignore("Action bar bugged on CI emulator")
```

Comment:
Can you reference a ticket here? Who knows if/how this can be fixed or is it just flaky and the test needs to be stabalized?